msvc/sdk-build.targets

```diff
@@ -64,7 +64,15 @@
       <PackageFile Include="@(_WinMDFileNames->Distinct())" Condition="Exists('%(FullPath)')">
         <PackagePath>$(PackageContentBasePath)\%(Filename)%(Extension)</PackagePath>
       </PackageFile>
-    </ItemGroup>
+      
+      <!-- Package up .pri files associated with any .winmds -->
+      <_PriFileNames Include="$(ProjectPriFullPath)">
+      </_PriFileNames>
+      <PackageFile Include="@(_PriFileNames)" Condition="Exists('%(RootDir)%(Directory)%(Filename).winmd')">
```

Comment:
No, there's no such scenario.  I don't understand what's wrong with this; we only want to grab .pri files that have associated .winmds.  If you have a better solution, please share it.

---
In reply to: [108994152](https://github.com/Microsoft/WinObjC/pull/2372#discussion_r108994152) [](ancestors = 108994152)